src/main/java/org/jenkinsci/plugins/ewm/steps/ExwsStep.java

```diff
@@ -45,6 +46,10 @@ public DescriptorImpl getDescriptor() {
 
```

Comment:
Add `@Symbol` above?

Reply:
I added `@Symbol("exws")`, was this the right way to do it?

Reply:
I'm a bit confused, wasn't discussed previously that we don't necessarily need `@Symbol`?

> @imaffe Symbols should not be needed for Pipeline steps, because you already have `getFunctionName()`. If you want to introduce a symbol there, I believe t should have the same value. Maybe it's better to do it in a separate PR anyway

Reply:
The documentation of the `Structs` plugin where `@Symbol` comes from is not clear to me. I find nothing that says the `getFunctionName()` can be used instead of the annotation.
 * [Wiki](https://wiki.jenkins.io/display/JENKINS/Structs+plugin)
* [README](https://github.com/jenkinsci/structs-plugin)
* [Symbol.java](https://github.com/jenkinsci/structs-plugin/blob/master/annotation/src/main/java/org/jenkinsci/Symbol.java)
* [Javadoc](https://javadoc.jenkins.io/plugin/structs/) => does not even mention `@Symbol`

Reply:
@oleg-nenashev We are confused. At first you said we do not need `@Symbol`, then you said to add it. Can you please clarify?

Reply:
Maybe `exwsConfiguration` or so, to be decided by @martinda @imaffe @alexsomai

Reply:
Thanks @oleg-nenashev for this clarification. To summarize, the `@Symbol` is used by the JCasC plugin to name the map in the YAML file. If no `@Symbol` is used, then the YAML syntax stays as it is:
```
unclassified:
  exwsAllocateStep:
...
```
If the `@Symbol("exwsGlobalConfiguration")` is used, then the YAML would look like:
```
unclassified:
  exwsGlobalConfiguration:
...
```
The name of the step in the Jenkinsfile Scripted Pipeline remains determined by the value returned by `getFunctionName()` in the descriptor. It is a bit hard to wrap my head around the fact that the step definition classes contain a descriptor that is used for the global configuration. I guess that's the Jenkins architecture.

Reply:
@imaffe @alexsomai I would recommend using `@Symbol(exwsGlobalConfiguration)`. But I am also okay with `@Symbol(exwsConfiguration)`.

Reply:
@martinda I agree with you, `@Symbol(exwsGlobalConfiguration)` seems more appropriate.

Reply:
Yes the step configuration going into global can be quite confusing,
Quite a few plugins separate that out to make it easier to manage

some examples:
sonarqube
slack
azure-keyvault

(not suggesting to do that now though)